docs/lib/datastore/fragments/native_common/setup-auth-rules.md

```diff
@@ -0,0 +1,123 @@
+Amplify gives you the ability to limit which individuals or groups should have access to create, read, update, or delete data on your various schema types by specifying an `@auth` directive.
+
+The following is a high level overview of the various authorization scenarios we support in the Amplify libraries.  Each scenario has a variety of options you can tune to fit the needs of your application.
+
+* [**Owner Based Authorization**](#owner-based-authorization): Uses Cognito User Pools as an auth provider to add ownership and access rules to instances of your model
+* [**Static Group Authorization**](#static-group-authorization): Uses Cognito User Pools as an auth provider with groups to apply access rules to instances of your model
```

Comment:
Same comment as above: "Limit an object instance's access to a specific group of users and define authorization rules for that group. Backend by Cognito User Pool."